test/e2e/network_test.go

```diff
@@ -839,12 +843,11 @@ func (tc *testContext) waitUntilJobSucceeds(name string) (string, error) {
 		if err != nil {
 			return "", err
 		}
-		labelSelector = "job-name=" + job.Name
-		tc.writePodLogs(labelSelector)
 		if job.Status.Succeeded > 0 {
-			logs, err := tc.getLogs(labelSelector)
+			labelSelector = "job-name=" + job.Name
```

Comment:
should this be set outside the `if` as it was before? otherwise, in the case we hit our retryCount, the `gatherPodLogs()` outside the loop will be fed an empty string for a labelSelector